guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc

```diff
@@ -31,73 +31,60 @@ The following tables indicate the destination port and the direction of network
 [cols="24%,20%,18%,38%",options="header"]
 |====
 | Port | Protocol | Service | Required For
+| 443 | TCP | HTTPS | {SmartProxy} to {Project}
+ifdef::katello[]
 | 5646   | TCP   |  amqp   |  {SmartProxy}'s Qpid dispatch router to Qpid dispatch router in {Project}
+endif::[]
 |====
 
 ifdef::foreman-el,katello[]
 Some of the following ports apply only to deployments that use the Katello plug-in.
 endif::[]
 
-ifdef::foreman-deb[]
-Some of the following ports apply only to deployments that use the Katello plug-in.
-For Debian deployments, ignore references to the Katello plug-in.
-endif::[]
-
 .Ports for Client to {SmartProxy} Communication
 [cols="24%,20%,18%,38%",options="header"]
 |====
-ifdef::satellite[]
 |Port |Protocol |Service |Required for
-|80 |TCP |HTTP |Anaconda, yum, and for obtaining Katello certificate
-updates
-|443 |TCP |HTTPS |Anaconda, yum, Telemetry Services, and Puppet
-|5647 |TCP |AMQP |Katello agent to communicate with {SmartProxy}'s
-Qpid dispatch router
-|8000 |TCP |HTTPS |Anaconda to download kickstart templates to hosts,
-and for downloading iPXE firmware
+|53 | TCP and UDP | DNS | Client DNS queries to a {SmartProxy}'s DNS service (Optional)
+|67 | UDP | DHCP | Client to {SmartProxy} broadcasts, DHCP broadcasts for Client provisioning from a {SmartProxy} (Optional)
+|69 | UDP |TFTP | Clients downloading PXE boot image files from a {SmartProxy} for provisioning (Optional)
+ifdef::katello,satellite[]
+|80 | TCP | HTTP |Content (like RPMs), and for obtaining Katello certificate updates
+|443 | TCP |HTTPS |Content (like RPMs)
+|5647 |TCP |AMQP |Katello agent to communicate with {SmartProxy}'s Qpid dispatch router (Optional)
+|8000 |TCP |HTTPS |Operating System installers like Anaconda to download installation templates to hosts, and for downloading iPXE firmware (Optional)
+|8140 |TCP |HTTPS |Puppet agent to Puppet server connections (Optional)
```

Comment:
Is this one only true for Katello because Foreman does not set Puppet up by default?

Reply:
No, it's in both. I duplicated it so the list of ports remained sorted. Doing that with a lot of if statements was longer than this. Once we align Katello on port 8443 for smart-proxy it'll be easier since everything will be complementary.